draft-ietf-httpapi-ratelimit-headers.md

```diff
@@ -244,7 +244,7 @@ This field can be sent in a trailer section.
 
 ## RateLimit-Remaining {#ratelimit-remaining-field}
 
-The "RateLimit-Remaining" field response field indicates the remaining quota units defined in {{service-limit}} associated with the client.
+The "RateLimit-Remaining" response field indicates the remaining quota units associated to the expiring-limit.
```

Comment:
associated _with_

Reply:
I know, but I always forget :) Thank you Mark.

```suggestion
The "RateLimit-Remaining" response field indicates the remaining quota units associated with the expiring-limit.
```